_health-care/_js/components/financial-assessment/ChildInformationSection.jsx

```diff
@@ -139,7 +139,7 @@ class ChildInformationSection extends React.Component {
       );
     }
 
-    if (this.props.data.sectionComplete) {
+    if (this.props.sectionIsCompleted) {
```

Comment:
I think the convention for naming things that return `true` or `false` is prefixing it with `is`: `isSectionComplete`